src/rpm/systemd-rpm-helper.in

```diff
@@ -0,0 +1,108 @@
+#!/bin/bash
+set -eu
+set -o pipefail
+
+command="${1:?}"
+shift
+
+command -v systemctl >/dev/null || exit 0
+
+case "$command" in
+    install-system-units)
+        systemctl --no-reload preset "$@"
+        ;;
+
+    install-user-units)
+        systemctl --no-reload preset --global "$@"
+        ;;
+
+    remove-system-units)
+        if [ -d /run/systemd/system ]; then
+            systemctl --no-reload disable --now "$@"
+        else
+            systemctl --no-reload disable "$@"
+        fi
+        ;;
+
+    remove-user-units)
+        systemctl --global disable "$@"
+
+        [ -d /run/systemd/system ] || exit 0
+
+        users=$(systemctl list-units 'user@*' --legend=no | sed -n -r 's/.*user@([0-9]+).service.*/\1/p')
+        for user in $users; do
+            SYSTEMD_BUS_TIMEOUT={{RPM_USER_MANAGER_TIMEOUT}} \
+                    systemctl --user -M "$user@" disable --now "$@" &
```

Comment:
This command gives me `Cannot access user instance remotely.`. Does it need some prerequisite patches?

Reply:
Yeah, https://github.com/systemd/systemd/pull/20157, which is only in `main` so far.